internal/testing/e2e/consul.go

```diff
@@ -287,6 +292,13 @@ func consulDeployment(namespace string, httpsPort, grpcPort int) *apps.Deploymen
 	labels := map[string]string{
 		"deployment": "consul-test-server",
 	}
+	var env []core.EnvVar
+	if consulEntLicense != "" {
```

Comment:
Just wondering, since the exercise of namespace code is going to require figuring out if we're running an enterprise v. non-enterprise version of Consul, should we have something to distinguish between the two? My thoughts are either an appended `ent` build tag on the test run with some `ent`-specific testing code or an environment variable we set to determine "this is an enterprise run". I'm assuming that this license env variable is set regardless of the consul image being run?

Reply:
That is correct about the license being present in CI during oss and ent runs.

I was going to play with writing ent tests in a file with a separate build tag and working out a way to include the tag during the ent jobs.